xml/System.Net.Sockets/TcpClient.xml

```diff
@@ -523,7 +523,9 @@ The `Available` property is a way to determine whether data is queued for readin
  This method does not block until the operation completes. To block until the operation completes, use one of the <xref:System.Net.Sockets.TcpClient.Connect%2A> method overloads.  
   
  For detailed information about using the asynchronous programming model, see [Calling Synchronous Methods Asynchronously](/dotnet/standard/asynchronous-programming-patterns/calling-synchronous-methods-asynchronously).  
-  
+
+> [!NOTE]
+> If you receive NotSupportedException with message "This protocol version is not supported" while using IPv6 address, then make sure you enabled IPv6 in constructor by passing <xref:System.Net.Sockets.AddressFamily.InterNetworkV6>.
```

Comment:
nit: Maybe we should consider do `This protocol version is not supported` instead of "This protocol version is not supported" to make it conspicuous.